canonical-web-and-design/snapcraft.io
# BG, BN, and RO buttons to publicise

## Done

- Added BG, BN, and RO buttons from https://github.com/snapcore/snap-store-badges/releases/tag/v1.4.1 thanks to @rares45 and @t4saha for the contributions

## Issue / Card

Fixes #2839

## QA

- Pull the branch
- Run the site using the command `./run`
- View the site locally in your web browser at: http://0.0.0.0:8004/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Visit the publicise page for any snap and ensure the new languages are present.

## Screenshots

![image](https://user-images.githubusercontent.com/479384/83641983-2a3cc000-a5a6-11ea-9137-56f6ab09c9f8.png)
![image](https://user-images.githubusercontent.com/479384/83642011-332d9180-a5a6-11ea-881a-fb73d7129753.png)
![image](https://user-images.githubusercontent.com/479384/83642051-3d4f9000-a5a6-11ea-8758-972210c863eb.png)
